internal/controller/argocd/deployment.go

```diff
@@ -895,6 +895,14 @@ func (r *ReconcileArgoCD) reconcileRepoDeployment(cr *argoproj.ArgoCD, useTLSFor
 		deploy.Spec.Template.Spec.InitContainers = append(deploy.Spec.Template.Spec.InitContainers, cr.Spec.Repo.InitContainers...)
 	}
 
+	// If the user has specified a custom volume mount that overrides the existing /tmp mount, then we should use the user's custom mount, rather than the default.
+	volumeMountOverridesTmpVolume := false
+	for _, volumeMount := range cr.Spec.Repo.VolumeMounts {
+		if volumeMount.MountPath == "/tmp" {
+			volumeMountOverridesTmpVolume = true
+		}
```

Comment:
```suggestion
			volumeMountOverridesTmpVolume = true
			break
		}
```
Thoughts on adding a break to exit early. The list of volumes won't be that long so it shouldn't make much difference even if we don't exit early.

Reply:
Done :heavy_check_mark: